.golangci.example.yml

```diff
@@ -66,8 +66,8 @@ run:
 
   # Define the Go version limit.
   # Mainly related to generics support in go1.18.
-  # Default: 1.17
-  go: 1.18
+  # Default: use Go version from the go.mod file, fallback on the env var `GOVERSION`, fallback on 1.17
```

Comment:
Should we follow the `go.mod` tooling and fallback on `Go` `1.16` if the `version` directive is missing?

> As of the Go 1.17 release, if the go directive is missing, go 1.16 is assumed.

https://go.dev/ref/mod#go-mod-file-go